test/int/project_test.go

```diff
@@ -598,6 +598,7 @@ var _ = Describe("AtlasProject", Label("int", "AtlasProject"), func() {
 					return resources.CheckCondition(k8sClient, createdProject, api.TrueCondition(api.ReadyType), validateNoErrorsMaintenanceWindowDuringUpdate(g))
 				}).WithTimeout(ProjectCreationTimeout).WithPolling(interval).Should(BeTrue())
 
+				time.Sleep(10 * time.Second)
```

Comment:
Why a sleep? why 10 seconds?

Reply:
The time Atlas took to return the expected response. Otherwise, I would have to refactor many things on the test a lot to make `checkMaintenanceWindowInAtlas()` work with `Eventually()`

Reply:
please leave TODO, and a jira ticket. This is real tech debt.